trade-finance-manager-api/src/v1/controllers/deal.party-db.js

```diff
@@ -60,7 +77,7 @@ const addPartyUrns = async (deal, auditDetails) => {
       ...deal.tfm,
       parties: {
         exporter: {
-          partyUrn: await getPartyUrn({ companyRegNo: deal.exporter.companiesHouseRegistrationNumber }),
+          partyUrn: await getPartyUrn({ companyRegNo: deal.exporter.companiesHouseRegistrationNumber, companyName: deal.exporter.companyName }),
```

Comment:
Perhaps we can declare these as constants on the top and simply refer to `companyRegNo` when making this call.